client/components/AdminFormExtension.jsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import { TextField, DatePicker, SelectField } from 'redux-form-material-ui';
+import { MenuItem, RaisedButton } from 'material-ui';
+import { Field } from 'redux-form';
+
+export default (props) => {
+
+  const style = {
```

Comment:
Can we get these styles into a separate file so they're not just defined inside the component? Material-UI is such a blessing and a curse.